flexget/plugins/list/sonarr_list.py

```diff
@@ -29,7 +29,13 @@ class SonarrSet(MutableSet):
             'include_data': {'type': 'boolean', 'default': False},
             'search_missing_episodes': {'type': 'boolean', 'default': True},
             'ignore_episodes_without_files': {'type': 'boolean', 'default': False},
-            'ignore_episodes_with_files': {'type': 'boolean', 'default': False}
+            'ignore_episodes_with_files': {'type': 'boolean', 'default': False},
+            'profile_id': {'type': 'integer', 'default': 1},
+            'season_folder': {'type': 'boolean', 'default': False},
+            'monitored': {'type': 'boolean', 'default': True},
+            'root_folder_path': {'type': 'integer', 'default': 1}, # Users are passing actual rootFolderPath ID (number (integer) starting from 1 and going up) as apposed to an number (integer) system starting from 0. Passing root_folder_path as a string seems to make the script fail, so that's why it's coded as an integerinstead. Read comment on show['rootFolderPath'] for more information.
```

Comment:
That's way too long, keep line limit to 108 chars please

Reply:
Okay, I'll try to condense it down in some way.

Reply:
Feel free to use multi line comments